@uportal/esco-content-menu/src/components/ContentFavorites.vue

```diff
@@ -165,16 +183,25 @@ export default {
       return url;
     },
     isXs() {
```

Comment:
Is `isXs` still reflecting what this does?
Or is thing something like a `resize` or `adjustSize` now?

Reply:
Good catch I didn't have eyes on whereas I would renamed it.